tracer_transport_test.go

```diff
@@ -592,7 +593,7 @@ var _ = Describe("Tracer Transports", func() {
 				Expect(context).To(BeEquivalentTo(knownContext3))
 			})
 
-			It("should drop higher 64 bits of a 128 bit TraceID", func() {
+			It("should error with an invalid TraceID", func() {
```

Comment:
Nit: Might be adding a couple words in the description about what makes this ID invalid. At a glance, it looks like it should be fine, and I think is actually the same ID used in a "valid" test above.

Reply:
I'll update it, the ID is 31 characters instead of 32 as the one above.